server/queues/tasks/DetachDraftsFromCollectionTask.ts

```diff
@@ -0,0 +1,48 @@
+import { Op } from "sequelize";
+import documentMover from "@server/commands/documentMover";
+import { sequelize } from "@server/database/sequelize";
+import { Collection, Document, User } from "@server/models";
+import BaseTask from "./BaseTask";
+
+type Task = {
+  collectionId: string;
+  actorId: string;
+  ip: string;
+};
+
+export default class DetachDraftsFromCollectionTask extends BaseTask<Task> {
+  async perform(task: Task) {
+    const [collection, actor] = await Promise.all([
+      Collection.findByPk(task.collectionId, {
+        paranoid: false,
+      }),
+      User.findByPk(task.actorId),
+    ]);
+
+    if (!actor || !collection || !collection.deletedAt) {
+      return;
+    }
+
+    const documents = await Document.findAll({
```

Comment:
```suggestion
    const documents = await Document.scope("withDrafts").findAll({
```

I believe you need to use the scope here or it will never find anything, drafts are filtered out of find operations by default.

Reply:
Gets overridden? [Here](https://github.com/outline/outline/blob/main/server/routes/api/documents/documents.ts#L356-L358) is another occurrence.

Reply:
Added scope nonetheless. No harm I suppose.